variable/string.cpp

```diff
@@ -3,6 +3,7 @@
 /// @file
 /// @author Simon Heybrock
 #include <chrono>
+#include <iomanip>
```

Comment:
Why is this needed?

Reply:
Because I removed some std headers from `unit.h` and this did not compile any more.